src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/Simulators/InstallCommand.cs

```diff
@@ -121,57 +121,96 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
 
     private async Task<bool> Install(Simulator simulator)
     {
-        if (simulator.Source is null)
+        var rawXcodeVersion = await GetXcodeVersion();
+        var xcodeVersion = Version.Parse(rawXcodeVersion);
```

Comment:
nit: all the callers of GetXcodeVersion() duplicate the Version.Parse so we could just return a `Version` from it directly.